### PR TITLE
[processor/tailsampling] Add new policies to composite policy

### PR DESCRIPTION
**Pull request by mapno**
**Description:** <Describe what has changed.>

Extend composite sub-policy option list to include the latest added policies. The following policies were missing from the composite sub-policy list:

- Latency
- StatusCode
- Probabilistic

### CODE REVIEW

**Comment by jpkrohling** [comment]
Note to self: compare with #6910 and check for conflicts.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 - `ecstaskobserver`: Fix "Incorrect conversion between integer types" security issue (#6939)
 - Fix typo in "direction" metrics attribute description (#6949)
+- Add support for new policies as composite sub-policies in tail sampling processor (#6975)
```
Comment (jpkrohling) [inline CHANGELOG.md:28]:
```suggestion
- `tailsamplingprocessor`: Add support for new policies as composite sub-policies (#6975)
```

**Review by jpkrohling — APPROVED** [review]
LGTM. An improvement could be made for the changelog, but it's not required.

**Comment by mapno** [comment]
Changed. Thanks for the quick review!

**Review by jpkrohling — APPROVED** [review]

**Comment by jpkrohling** [comment]
I restarted the load tests, which are probably unrelated to this PR. If I forget about this, feel free to ping me in order to get this merged.

**Pull request merged**